BestBuyCRUD/EmployeesRepo.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using MySql.Data.MySqlClient;
+
+namespace BestBuyCRUD
+{
+    class EmployeesRepo
+    {
+        public string connectionString; // field
+
+        public EmployeesRepo(string conn)
+        {
+            connectionString = conn;
+        }
+
+        public List<Employees> GetEmployees()
+        {
+            MySqlConnection connect = new MySqlConnection(connectionString);
+
+            using (connect)
+            {
+                connect.Open();
+
+                MySqlCommand cmd = connect.CreateCommand();
+                cmd.CommandText = "Select EmployeeId AS ID, FirstName AS First, MiddleInitial AS Middle, LastName AS Last," +
```

Comment:
Small note: it's probably best to put the SQL keywords in all uppercase, such as "SELECT" instead of "Select"